improver_tests/calibration/dz_rescaling/test_apply_dz_rescaling.py

```diff
@@ -50,61 +52,29 @@
 
 
 def _create_forecasts(
-    forecast_reference_time: str,
-    validity_time: str,
-    forecast_period: float,
-    forecast_percs: List[float],
+    forecast_reference_time: str, validity_time: str, forecast_percs: List[float],
 ) -> Cube:
     """Create site forecast cube for testing.
 
     Args:
         forecast_reference_time: Timestamp e.g. "20170101T0000Z".
         validity_time: Timestamp e.g. "20170101T0600Z".
-        forecast_period: Forecast period in hours.
         forecast_percs: Forecast wind speed at 10th, 50th and 90th percentile.
 
     Returns:
         Forecast cube containing three percentiles and two sites.
     """
-    data = np.array(forecast_percs).repeat(2).reshape(3, 2)
+    data = np.array(forecast_percs, dtype=np.float32).repeat(2).reshape(3, 2)
+    percentiles = [10, 50, 90]
 
-    perc_coord = DimCoord(
-        np.array([10, 50, 90], dtype=np.float32), long_name="percentile", units="%",
-    )
-    fp_coord = AuxCoord(
-        np.array(
-            forecast_period * SECONDS_IN_HOUR,
-            dtype=TIME_COORDS["forecast_period"].dtype,
-        ),
-        "forecast_period",
-        units=TIME_COORDS["forecast_period"].units,
-    )
-    time_coord = AuxCoord(
-        np.array(
-            pd.Timestamp(validity_time).timestamp(), dtype=TIME_COORDS["time"].dtype,
-        ),
-        "time",
-        units=TIME_COORDS["time"].units,
-    )
-    frt_coord = AuxCoord(
-        np.array(
-            pd.Timestamp(forecast_reference_time).timestamp(),
-            dtype=TIME_COORDS["forecast_reference_time"].dtype,
-        ),
-        "forecast_reference_time",
-        units=TIME_COORDS["forecast_reference_time"].units,
-    )
-
-    cube = build_spotdata_cube(
+    cube = set_up_spot_percentile_cube(
         data,
-        "wind_speed_at_10m",
-        "m s-1",
-        altitude,
-        latitude,
-        longitude,
-        wmo_id,
-        scalar_coords=[fp_coord, time_coord, frt_coord],
-        additional_dims=[perc_coord],
+        percentiles,
+        name="wind_speed_at_10m",
+        units="m s-1",
+        wmo_ids=["00001", "00002"],
```

Comment:
Should this use the global `wmo_id` variable? (It would probably be better if this variable was capitalised.) I think that the `altitude`, `latitude` and `longitude` global variables could also potentially be moved to being defined in the `create_scaling_factor_cube` now.

Reply:
All done.